src/index.js

```diff
@@ -1073,6 +1082,7 @@ export default class Carousel extends React.Component {
           getSliderStyles(this.props.width, this.props.height),
           this.props.style
         )}
+        dir={rtl ? 'ltr' : 'auto'}
```

Comment:
where does `ltr` come in? and `auto`? it doesn't look like `dir` is being used anywhere

Reply:
As you well say, `dir` attribute is not used anywhere.
According to https://www.w3.org/International/questions/qa-html-dir 

> In HTML the base direction is either set explicitly by the nearest parent element that uses the dir attribute, or, in the absence of such an attribute, the base direction is inherited from the default direction of the document, which is left-to-right (LTR).

This means that the application of `dir="rtl"` to an ancestor element of the `<Carousel />` is inherited from the `<Carousel />`, causing a wrong rendering of `.slider-list` and its controls.

The proposed modification is intended to overwrite the `dir` attribute of the `.slider` element assuming that if a flipped version is required (providing the prop `rtl` to the `<Carousel />`) an ancestor element could have set the attribute `dir="ltr"`.